scripts/reindex.py

```diff
@@ -116,6 +121,13 @@ def main(argv: List[str]):
             azul_client.remote_reindex(args.partition_prefix_length)
         else:
             azul_client.reindex()
+        if args.wait:
+            logger.info('Waiting for queues to empty ... ')
+            while True:
+                sleep(5)
+                total_message_count = Queues.count_messages(queues)
```

Comment:
If you have an instance you should call through the instance because that avoids hard-coding the class-name again. However, see overall comments.